python/ray/experimental/sgd/tf/examples/tf-example-sgd.yaml

```diff
@@ -47,13 +47,16 @@ worker_nodes:
     #         MarketType: spot
 
 setup_commands:
-    - conda install setuptools=41.0.1=py36_0 wrapt=1.11.2 --yes # workaround to fix wrapt error
-    - ray || pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.9.0.dev0-cp36-cp36m-manylinux1_x86_64.whl
-    - pip install -U ipdb ray[rllib]
-    - pip install tensorflow==2.0.0-rc0
-
-file_mounts: {
-}
+    - conda install setuptools=45.1.0=py36_0 wrapt=1.11.2 --yes # workaround to fix wrapt error
+    - ray &> /dev/null || pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.9.0.dev0-cp36-cp36m-manylinux1_x86_64.whl
+    - pip install -U ray[tune]
+    - pip install tensorflow==2.1.0
+    # todo: git clone distribution token
+
+file_mounts:
+    "~/anaconda3/lib/python3.6/site-packages/ray/experimental/sgd/pytorch/": "../../pytorch/"
+    "~/anaconda3/lib/python3.6/site-packages/ray/experimental/sgd/tf/": "../../tf/"
+    "~/anaconda3/lib/python3.6/site-packages/ray/experimental/sgd/examples/": "."
```

Comment:
Don't forget to revert these changes for filemounts on site-packages

Reply:
I removed them in a previous commit and pushed them back in by accident, my bad